src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -1130,6 +1118,13 @@
 				}
 			});
 
+			hostMessaging.onMessage('title', async (_event, data) => {
```

Comment:
this is `title` and I noticed below it's `'set-title': string | undefined;` for `ToWebviewMessage`. Those are not referring to the same thing, right? Don't know anything about this code, but just wanted to make sure.